lib/iris/fileformats/grib/grib_phenom_translation.py

```diff
@@ -190,10 +190,10 @@ def _make_grib2_cf_entry(param_discipline, param_category, param_number,
                               '(skipping).'.format(standard_name))
                 return None
         # convert units string to iris Unit (i.e. mainly, check it is good)
-        iris_units = iris.unit.Unit(units)
+        cf_unitss = cf_units.Unit(units)
```

Comment:
Yeah looks like a find and replace for "iris.unit" gone wrong, since as a regex this matches "iris_unit" also.